tests/compatibility-test

```diff
@@ -0,0 +1,84 @@
+#!/bin/sh
+
+trap '[ "$?" -ne 0 ] && printf "\nAn error occurred\n"' EXIT
+
+verbose=0
+image_index=0
+# Dont' run this command as sudo.
+if [ "$(id -u)" -eq 0 ]; then
+	printf >&2 "Running %s as sudo is not supported.\n" "${0}"
+	printf >&2 "Please check the documentation on:\n"
+	printf >&2 "\tman distrobox-compatibility\t"
+	printf >&2 "or consult the documentation page on:\n"
+	printf >&2 "\thttps://github.com/89luca89/distrobox/blob/main/docs/compatibility.md\n"
+	exit 1
+fi
+
+# Print usage to stdout.
+# Arguments:
+#   None
+# Outputs:
+#   print usage with examples.
+show_help() {
+	cat <<EOF
+Usage:
+
+	compatibility-test [--index X]
+
+Options:
+
+	--index/i:		from which index image to start
+	--help/-h:		show this message
+	--verbose/-v:		show more verbosity
+
+EOF
+}
+
+# Parse arguments
+while :; do
+	case $1 in
+	-h | --help)
+		# Call a "show_help" function to display a synopsis, then exit.
+		show_help
+		exit 0
+		;;
+	-v | --verbose)
+		shift
+		verbose=1
+		;;
+	-i | --index)
+		if [ -n "$2" ]; then
+			image_index="$2"
+			shift
+			shift
+		fi
+		;;
+	*) # Default case: If no more options then break out of the loop.
+		break ;;
+	esac
+done
+
+set -o errexit
+set -o nounset
+# set verbosity
+if [ "${verbose}" -ne 0 ]; then
+	set -o xtrace
+fi
+
+image_list=$(grep -E "docker.io|quay.io|ghcr|registry.|ecr." "$(dirname "${0}")"/../docs/compatibility.md | cut -d'|' -f 4 | sed 's/<br>/\n/g' | tr -d ' ' | tail -n +2)
+image_total=$(grep -E "docker.io|quay.io|ghcr|registry.|ecr." "$(dirname "${0}")"/../docs/compatibility.md | cut -d'|' -f 4 | sed 's/<br>/\n/g' | tr -d ' ' | tail -n +2 | wc -l)
```

Comment:
`image_total=$(echo "$image_list" | wc -l)`

would likely save you some cycles :)

Reply:
he he you're right :smile: